example/example-metadata.ttl

```diff
@@ -100,230 +211,99 @@
 
```

Comment:
How did the roles here come into being? I'm pretty sure `<beol-gehr>` and `<beol-nagel>` would consider themselves "editor" too.

Furthermore, I think Fabienne Schwizer should be listet here too.

Reply:
Also, things like "Teststrasse" should probably be replaced by real information. (Though I'm wondering to what extent the address should really be here? Privacy concerns and such...)